language/gpt-oss/eval_mlperf_performance.py

```diff
@@ -0,0 +1,645 @@
+#!/usr/bin/env python3
```

Comment:
This script is for analysis (i.e. not on the critical path)? recommend putting into sub dir (like reports/ or utils/)